dotcom-rendering/src/components/Slideshow.tsx

```diff
@@ -136,96 +89,115 @@ const captionStyles = css`
 		rgba(0, 0, 0, 0) 0%,
 		rgba(0, 0, 0, 0.8) 100%
 	);
-	color: ${neutral[100]};
-	padding: 60px 8px 8px;
+	color: ${palette.neutral[100]};
+	padding: 60px ${space[2]}px ${space[2]}px;
 `;
 
 const additionalDynamoCaptionStyles = css`
 	${from.tablet} {
 		top: 0;
 		bottom: initial;
-		padding-top: 8px;
+		padding-top: ${space[2]}px;
 		background: linear-gradient(
-			to bottom,
-			rgba(0, 0, 0, 0.8) 0%,
-			rgba(0, 0, 0, 0) 100%
+			to top,
+			rgba(0, 0, 0, 0) 0%,
+			rgba(0, 0, 0, 0.8) 100%
 		);
 	}
 `;
 
 /**
- * **Slildeshow**
+ * Slideshow
+ * =========
  *
- * Uses the perpetual canon effect to create a slideshow of images using only css. It:
+ * Infinitely display a series of slides using CSS animations only,
+ * using a technique similar to a musical “round” or “perpetual canon”.
  *
  * - overlays images over each other so they are stacked
  * - positions and styles the given caption values
- * - adds an animation effect to each image such that it only has opacity 1 for a short period
- * - uses animation-delay to stagger the start for each image's animation, thus
- *   causing the slideshow effect
+ * - adds an animation effect to images such that they are only opaque for a short period
+ * - uses a staggered delay to the animation which creates the slideshow effect
+ *
+ *  ```
+ *   image 1 : ████████████████████████████ (not animated, always visible)
+ *
+ *   image 2 : ▁▁▁▁▁▃▅███████▁▁▁▁▁▁▁▁▁▁▁▁▁▁ (fades in, off after 3 is fully opaque)
+ *
+ *   image 3 : ▁▁▁▁▁▁▁▁▁▁▁▁▃▅███████▁▁▁▁▁▁▁ (fades in, off after 4 is fully opaque)
  *
- *                 time ->
- *   image-1 > ####|----|----|----
- *   image-2 > ----|####|----|----
- *   image-3 > ----|----|####|----
- *   image-4 > ----|----|----|####
+ *   image 4 : ▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▃▅█████▅▃ (fades in and out)
+ *       time →     →     →     →     →
+ *  ```
```

Comment:
Really cool description!

Reply:
![Graphic Design is My Passion](https://www.graphicscycle.com/wp-content/uploads/2023/03/Graphic-Design-is-My-Passion-Meme.webp)